src/aksharamukha/index.ts

```diff
@@ -144,6 +160,7 @@ export default class Aksharamukha {
 			postOptions
 		}: ProcessProps = defaultProcessProps
 	) {
+		const pyodide = this.pyodide ?? (this.pyodide = await Aksharamukha.getPyodide());
```

Comment:
I think this code is technically unreachable, considering a fresh instance of Aksharamukha would always have a Pyodide instance attached to it.